_specs/ecip-1095.md

```diff
@@ -0,0 +1,74 @@
+---
+lang: en
+ecip: 1095
+title: Change the ETC Proof of Work Algorithm to Sha-3-256
+status: Draft
+type: Standards Track
+category: Core
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/342
+author: Stevan Lohja (stevanio@protonmail.com)
+created: 2020-08-20
+---
+
+# Change the ETC Proof of Work Algorithm to Sha-3-256 (vanilla Sha-3)
+
+__Abstract.__ Make Sha-3-256 Standard the Proof of Work algorithm for Ethereum Classic. Ethereum’s development coincided with the development of the Sha-3 standard, but the Sha-3 standard was finalized shortly after Ethereum launched. So, Ethereum was left with a variant of Sha-3 that did not produce standard Sha-3 hashes. This variant of Sha-3 utilized in Ethereum was named Keccak-256 to avoid confusion. Keccak-256 is utilized in the EVM, but the Proof of Work algorithm is Ethash. Ethash is a combination of Keccak256 & Daggerhashimoto which contributes to the memory-intensive feature (Dag). The Dag was added due to the premise of ASIC resistance, but that premise is flase since Ethash ASICs exist. So, Ethash’s Dag is built on the false premise of ASIC resistance and not utilizing the official Sha-3 Standard algorithm.
+
+Ethereum Classic's hash rate is also a minority of Ethereum's and the amount of rentable hash rate is available on the market to 51% attack the Ethereum Classic network. In summary, this proposal seeks to:
+
+- Remove Daggerhashimoto (Dag)
+- Bump Keccak-256 to Sha-3-256 Standard
+- Become a majority PoW algorithm in the industry
+
+__Disclaimer.__ This is an alternative proposal to ECIP-1049 [1] since Keccak-256 and _vanilla_ Sha-3 produce different hashing results. However, most of the arguments in this proposal are compatible with ECIP-1049 since Keccak-256 and _vanilla_ Sha-3 differ in padding. ECIP-1049 and this proposal are both in support of switching the PoW to a Sha-3-based algorithm.
+
+__Motivation.__ The motivation of this proposal is to standardize the cryptographic hash function of the Ethereum Classic network with _vanilla_ Sha-3 and become a majority chain in it's respective PoW.
+
+__Rationale.__ 
+
+- __Cryptographic standardization__: By bumping Keccak-256 to _vanilla_ Sha-3, Ethereum Classic will have the official Sha-3 algorithm recognized by NIST. Sha-3 features:
+  - __Enhanced security__: Sha-3 is the latest member of the Secure Hash Algorithm family of standards and certified by the Federal Information Processing Standards (FIPS) [2].
```

Comment:
One could argue that not having the final standard actually adds sovereignty and arguably a layer of security.